client/scenes/lobby_gui.cpp

```diff
@@ -607,17 +607,11 @@ void scenes::lobby::gui_settings()
 		const auto current = config.resolution_scale;
 		const auto width = stream_view.recommendedImageRectWidth;
 		const auto height = stream_view.recommendedImageRectHeight;
-		if (ImGui::BeginCombo(_S("Resolution scale"), fmt::format(_F("{}% - {}x{} per eye"), int(current * 100), int(width * current), int(height * current)).c_str()))
+		auto intScale = int(current * 100);
+		if (ImGui::SliderInt(_S("Resolution scale"), &intScale, 50, 500, fmt::format(_F("%d%% - {}x{} per eye"), int(width * current), int(height * current)).c_str()))
```

Comment:
Can you update the existing translations for the "{}" -> "%d" change

Reply:
Do you mean here or in other places? The %d gets formatted by ImGui, our _F function doesn't touch it.

Reply:
In client/locale there are translations, checked by https://github.com/WiVRn/WiVRn/actions/runs/12719300523/job/35467616628?pr=233

You just have to replace the key (msgid) and values (msgstr) in wivrn.po files.

Reply:
Ah, I see; I misunderstood. Fixed.